requirements.txt

```diff
@@ -1,3 +1,4 @@
 aiofiles<=23.1.0
 Jinja2==3.1.6
+python-telegram-bot-calendar==1.0.5
```

Comment:
Move the source code of the package to `hammett/widgets/calendar`. Any other suggestions are welcome.